src/components/Admin/ManagePromptsEntries.vue

```diff
@@ -12,7 +12,8 @@
     title="Manage Prompts & Entries"
   >
     <template v-slot:top-right>
-      <q-input data-test="input-search" debounce="300" dense placeholder="Search" v-model="filter">
+      <q-input debounce="300" dense placeholder="Search" v-model="filter"
+               :data-test="(promptStore.isLoading || entryStore.isLoading) ? '' : 'input-search'">
```

Comment:
Data test is added only if the load is finish so cypress test will wait until the load is finish

Reply:
Nice!!